src/databricks/labs/remorph/connections/database_manager.py

```diff
@@ -56,7 +56,25 @@ def _create_connector(db_type: str, config: dict[str, Any]) -> DatabaseConnector
 
 class SnowflakeConnector(_BaseConnector):
     def _connect(self) -> Engine:
-        raise NotImplementedError("Snowflake connector not implemented")
+        import snowflake.sqlalchemy  # type: ignore
```

Comment:
`pylint disable`

Reply:
There are two issues here. I added `# pylint: disable=import-outside-toplevel` to address the pylint error. However, `type: ignore` is still needed as the snowflake module is missing library stubs / py.typed marker and fails a `make fmt`.

Reply:
Make sense